Create table-missing-headers.md

1Copenut opened the pull request:
Adding a draft version of the first PDF issue detected in public-facing documents on the `www` domain

1Copenut committed:
Create table-missing-headers.md

Adding a draft version of the first PDF issue detected in public-facing documents on the `www` domain

1Copenut committed:
Adding linked images with alt text

1Copenut committed:
Removing the extra step for compliance check

The step to the confirmation step feels wrong with the embedded images.

1Copenut committed:
Adding a table of contents

gmaciolek committed:
a few minor changes to headings

- removed page title and table of contents from table of contents
- changed "Statement of issue" to "Description of issue"
- changed "Fix issue" to "Steps to fix issue"
- added "(preferred)" to "Define table headers in Microsoft Word"

gmaciolek commented on platform/accessibility/common-issues/pdf/table-missing-headers.md:
```diff
@@ -0,0 +1,76 @@
+## Description of the issue
+
+- Synopsis: Make tables accessible in PDF documents by using table elements. The applicable standard from Section508.22.h dictates to "associate data and header cells for multi-dimensional data tables."
+- Severity: Critical
```
Consider linking to severity rubric. If we're not already consistently referring to severity levels, we should do that. I think that elsewhere we use` 508-defect-0`, `508-defect-1`, etc when referring to severity levels? Maybe we should use that here as well?

1Copenut replied:
I think so, and start including document issues. I have an idea. Will add it to this PR.

gmaciolek commented on platform/accessibility/common-issues/pdf/table-missing-headers.md:
```diff
@@ -0,0 +1,76 @@
+## Definition of done
+
+- There SHOULD be **0 table header violations** in the Word [Check Accessibility panel](https://support.microsoft.com/en-us/office/improve-accessibility-with-the-accessibility-checker-a16f6de0-2f39-4a2b-8bd8-5ad801426c7f)
+- There MUST be  in the Acrobat [Accessibility Checker](https://www.adobe.com/accessibility/products/acrobat/using-acrobat-pro-accessibility-checker.html)
```
Seems like this line is missing something...?

1Copenut replied:
Yep, the bolded portion didn't get carried over. Fixing now.

gmaciolek commented on platform/accessibility/common-issues/pdf/table-missing-headers.md:
```diff
@@ -0,0 +1,76 @@
+
+## Steps to fix the issue
+
+Tables that are missing table headers can be remediated in Word or Acrobat. The preferred method illustrated here is to fix the issue in a Word document whenever possible. Instructions to fix a table header in Acrobat are included in the second documentation link.
```
If you want me to get nit-picky about details, I would change:
"Tables that are missing table headers can be remediated in Word or Acrobat. The preferred method illustrated here is to fix the issue in a Word document whenever possible. Instructions to fix a table header in Acrobat are included in the second documentation link."
to
"Remediate tables in Word (preferred) or Acrobat."

gmaciolek replied:
Also, this made me think that I should add something to the word list about how to refer to Word and Acrobat. (e.g., Microsoft Word vs. Word and Adobe Acrobat vs. Acrobat) No need to change anything in your doc at this time, but I'll follow up with you if we add guidance to the word list for this.

1Copenut replied:
Good call. I'll go ahead and add the company names because I've heard some questions about "Adobe" when I think the ask was about "Adobe Acrobat".

gmaciolek commented on platform/accessibility/common-issues/pdf/table-missing-headers.md:
```diff
@@ -0,0 +1,76 @@
+3. Click `<Table>` to expand the tree for the table you want to update
+4. Click the first TD icon to highlight the first cell in your table
+5. Click `<TD>` in the Tags inspector to edit the tag. Change it to a `<TH>` and press `ENTER`
+6. The tag icon will change slightly to a red cell inside a table. This confirms your cell has been updated to a table header.
```
This isn't really a step. I'd make it part of Step 5.

1Copenut committed:
Update table-missing-headers.md

Revising based on comments from Gretchen

1Copenut committed:
Update defect-severity-rubric.md

Adding a breakout for web apps and documents in severity 1 issues. Will add these breakouts for other severity levels as they become applicable.

1Copenut committed:
Update table-missing-headers.md

Adding a link to the severity rubric

gmaciolek commented on platform/accessibility/common-issues/pdf/table-missing-headers.md:
```diff
@@ -0,0 +1,75 @@
+
+### Define table headers in Microsoft Word
+
+Table header rows can be defined in a few steps:
```
I just realized I missed this before, but we do have recommendations in the style guide about [formatting multi-action procedures](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/documentation/style-guide/formatting-and-organization.md#procedures).

The main difference is to use **bold** instead of `code` We also try to avoid saying "Click" in favor of "Select"

I wouldn't delay publishing based on this, though, just a note for the future.

1Copenut replied:
I went ahead and updated them, so I have a good guide for the next 6-7 documents. Thank you @gmaciolek !

gmaciolek replied:
🎉  go team!

1Copenut committed:
Update table-missing-headers.md

Updating code blocks to bold blocks

1Copenut commented on platform/accessibility/common-issues/pdf/table-missing-headers.md:
```diff
@@ -0,0 +1,75 @@
+
+### Acrobat Pro accessibility tools
+
+Run the Acrobat Accessibility Checker by clicking the purple `Accessibility` button on the top right of your application window. Click `Accessibility Check` in the accessibility panel, then click the `Start Checking` button. This will run the accessibility checker, and report any issues in a panel on the left of your document.
```
Done!

1Copenut committed:
Update table-missing-headers.md

Swapping out the term "click" for "select"

1Copenut commented:
@gmaciolek I made the last few changes and I'm going to merge tomorrow morning unless you see any further tweaks. I'm about at EOD now, and will work on a few other things tonight. Thank you for the quick reviews and feedback.

emilywaggoner reviewed (approved):
Looks great yall!
